src/value/ser.rs

```diff
@@ -215,14 +222,13 @@ impl serde::Serializer for Serializer {
     }
 
     fn serialize_map(self, _len: Option<usize>) -> Result<Self::SerializeMap, Error> {
-        Ok(
-            SerializeMap {
-                map: Map::new(),
-                next_key: None,
-            },
-        )
+        Ok(SerializeMap::Map {
```

Comment:
Please revert unnecessary whitespace changes like this one. We can run the new rustfmt after the PR lands.

Reply:
Hmm, not sure how that happened!